core/src/main/java/com/rudderstack/android/sdk/core/TransformationResponseDeserializer.java

```diff
@@ -0,0 +1,44 @@
+package com.rudderstack.android.sdk.core;
```

Comment:
Do we need custom deserializer for this? What was the issue?

Reply:
This was needed to handle the case when `transformedBatch.payload.event` is `null`. LMK, if we can handle this case in any better way.